packages/models/types/index.d.ts

```diff
@@ -156,14 +156,16 @@ declare module '@appland/models' {
     readonly httpClientRequest?: HttpClientRequest;
     readonly httpClientResponse?: HttpClientResponse;
     readonly elapsedTime?: number;
-    readonly definedClass?: string | null;
-    readonly requestPath?: string | null;
-    readonly requestMethod?: string | null;
-    readonly route?: string | null;
-    readonly sqlQuery?: string | null;
+    readonly definedClass?: string;
+    readonly requestPath?: string;
+    readonly requestMethod?: string;
+    readonly requestContentType?: string;
+    readonly responseContentType?: string;
+    readonly route?: string;
+    readonly sqlQuery?: string;
     readonly fqid: string;
-    readonly previousSibling?: Event | null;
-    readonly nextSibling?: Event | null;
+    readonly previousSibling?: Event;
+    readonly nextSibling?: Event;
```

Comment:
These were correctly typed. The value returned is either `string | undefined | null` or `Event | undefined | null`.